CHANGELOG.md

```diff
@@ -4,23 +4,27 @@
 
 **Bug Fixes**
 
-* Due to inconsistent if-statements, FMA would sometimes write to a non-existent file unit and without opening the file properly.
+* Due to inconsistent if-statements, FMA would sometimes write to a non-existent file unit without opening the file properly.
 * DYNK would not build with nagfor due to two malformed strings.
-* Fixed building with `ROND_ZERO` flag. The `matlib_bouncer` was trying to call non-existent round-towards-zero routines for `exp_mb()` functions (which is handled by round down) and `log10_mb` which was missing for `ROUND_ZERO` and has been added to `CRLIBM`.
-* A variable for DYNK FIR/IIR functions were written to memory twice, but for one case with an invalid index. This has been removed.
-* Removed remaining `COLLIMAT` flag in the beam-gas module that prevented the `BEAMGAS` flag from building.
+* Fixed building with `ROUND_ZERO` flag. The `matlib_bouncer` was trying to call non-existent round-towards-zero routines for `exp_mb()` functions (which is handled by round down `exp_rd()`), and `log10_mb` which was missing for `ROUND_ZERO` and has been added to `CRLIBM`.
+* A variable for DYNK FIR/IIR functions was written to memory twice, but with an invalid index for one of those writes. This write has been removed.
+* Removed a remaining `COLLIMAT` flag in the beam-gas module that prevented the `BEAMGAS` flag from building.
```

Comment:
"Removed a remaining `COLLIMAT` flag in the" -> "Removed a remaining `#IFDEF COLLIMAT` in the" ?